autoprotocol/protocol.py

```diff
@@ -1469,7 +1469,7 @@ def equalize_lengths(
             # Update source volume
             source_location.add_volume(-total_volume_dispensed)
         device_mode_params = LiquidHandleBuilders.device_mode_params(
-            model=model, chip_material=chip_material, nozzle=nozzle
+            model=model, material=chip_material, nozzle=nozzle
```

Comment:
good point especially if we are not the only users of this repo

Reply:
true. I did however do a repgrep and updated all dependencies so that it's back compatible. Do you want me to just keep it? I have a preference for just `material` since we change it to `material` in the original code....